README.md

```diff
@@ -111,7 +111,7 @@ Crm::Contact.where(:locality, :equals, 'Boston').
 
 ## License
 
-Copyright (c) 2015 - 2020 [Infopark AG](https://infopark.com).
+Copyright (c) 2015 - 2020 [JustRelate](https://justrelate.com).
```

Comment:
Please update the copyright year to 2021 instead of 2020.

Reply:
Ah, in the copyright the company name must be specified fully: JustRelate Group GmbH